image_classification/ConvMixer/tests/test_onecyclelr.py

```diff
@@ -0,0 +1,55 @@
+#   Copyright (c) 2021 PPViT Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from ..utils import OneCycleLRScheduler
```

Comment:
`from ..utils`  can be modified to  `from utils`,  so that in  ConvMixer/ folder, we can use  command  `python -m  unittest` to performs the unit test.